src/components/PriceBar/TileColorBoard.tsx

```diff
@@ -103,11 +134,20 @@ const TileColorBoard: React.FC = () => {
             <Text
               fontSize={{ base: "md", lg: "lg" }}
               fontWeight={{ base: "700", lg: "800" }}
-              marginLeft="2px"
+              marginLeft="5px"
+              marginRight="20px"
             >
               ${useTotalPrice === "0.00" ? "Loading..." : useTotalPrice}
             </Text>
           </Center>
+          <Button
+            variant="shareBtn"
+            marginRight="30px"
+            onClick={handleAddToCart}
+            data-testid="add-to-cart-button"
+          >
+            Add to Cart
+          </Button>
```

Comment:
considering extracting the button/budget part to be a separate component instead of bloating the TileColorBoard component?